Marlin/ultralcd.cpp

```diff
@@ -1077,6 +1077,137 @@ void kill_screen(const char* lcd_msg) {
   #endif // BABYSTEPPING
 
   #if ENABLED(AUTO_BED_LEVELING_UBL)
+    #if ENABLED(AUTO_BED_LEVELING_UBL_GFX)
```

Comment:
This should use the same graphics as BABYSTEP_ZPROBE_GFX_OVERLAY, as duplicating them would use up a lot of memory. 

Where I have "#if ENABLED(BABYSTEP_ZPROBE_GFX_OVERLAY)", simply use "#if ENABLED(BABYSTEP_ZPROBE_GFX_OVERLAY) || ENABLED(AUTO_BED_LEVELING_UBL)